demo/26px/index.html

```diff
@@ -274,6 +282,14 @@ <h2 class="u-fs-lg u-mb">Wordmark Layout</h2>
             <use xlink:href="../index.svg#zd-svg-icon-26-wordmark-gather">
           </svg>
         </span
+        ><span class="l-grid__item u-1/2 u-mb u-truncate">
```

Comment:
Demo looks like it’s missing the logo-word layout here. As such, the column layout is messed up.